local/compose/convergence.go

```diff
@@ -60,7 +60,7 @@ func (s *composeService) ensureService(ctx context.Context, project *types.Proje
 		missing := scale - len(actual)
 		for i := 0; i < missing; i++ {
 			number := next + i
-			name := fmt.Sprintf("%s_%s_%d", project.Name, service.Name, number)
+			name := getContainerNamePrintLabel(project.Name, service, number)
```

Comment:
Need to validate container_name is not used with more than one container for a service